### DISCUSSION TITLE
请问 antd  autoheight 的 textarea 能实现垂直居中吗?

### DISCUSSION THREAD

**Discussion by vaynevayne** [ant-design/ant-design#45685] [Q&A]
如题

**Comment by kiner-tang** [comment]
可以参考这个：https://codesandbox.io/s/gua-ying-wen-ben-gao-du-de-wen-ben-yu-antd-5-11-0-forked-rs4ywr
<img width="734" alt="image" src="https://github.com/ant-design/ant-design/assets/10286961/87264cbf-f6c1-4698-b232-bd8e8ff50bff">

**Reply by vaynevayne** [reply]
<img width="543" alt="image" src="https://github.com/ant-design/ant-design/assets/124666577/f345def3-944a-481c-b0a0-1b3191833342">


发现个问题, 这里不应该写这个属性, 不然无法居中

**Reply by kiner-tang** [reply]
哪个属性？

**Reply by vaynevayne** [reply]
min-height

**Reply by kiner-tang** [reply]
你现在的期望是啥？上面的垂直居中方案没有满足你的需求吗

**Reply by vaynevayne** [reply]
1. 不要写 min-height, 我的css样式中的 height 失效了
2. sm 时,需要再次修改 min-height
<img width="1349" alt="image" src="https://github.com/ant-design/ant-design/assets/124666577/57319590-900d-4050-942b-f4e78a003904">

还说我想修改这个属性, 真麻烦,  花时间设计的token 系统, 真心难用, 不参考其他开源项目吗? 比如 emotine
,

**Reply by MadCcc** [reply]
min-height 为什么会让 height 失效？

**Reply by MadCcc** [reply]
想要改的少的话就改 `controlHeight` 和 `controlHeightSM` 这两个 token，`min-height` 取值来自这两个
